### ISSUE TITLE
[Programmatic Access - Bot Framework Web Chat - Card Productvideo]: Captions are not provided for video content under 'Card Productvideo' section.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5361]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
If Closed Captions and Subtitles are not defined for pre-recorded video controls present on the application, then users who have a hearing loss cannot understand and access the auditory information of the synchronized media content through captions hence the purpose of the video would not be clear, and user might fail to utilize the complete functionality of the video control.
### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/ 

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Productvideo word in type your message text box.
4. Navigate to the results.
5. Verify whether Captions are available for the Video content or not.

### What do you expect?

Captions should be provided for video content under the 'Card Productvideo' section.

### What actually happened?

Captions are not provided for video content under the 'Card Productvideo' section.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/b3a3f420-1adc-4ae2-bd5c-2fdc8bb4b30b)
https://github.com/user-attachments/assets/e42ddd71-1fe4-4730-8efc-7d2a4b911dbf

### MAS reference

MAS 1.2.2 - Captions (Pre-recorded)

### Note: 

- Same issue reproes for all the videos throughout the application.

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS1.2.2;#Win11-Edge(Chromium);

**Comment by OEvgeny** [comment]
This is due to adaptive cards v1.3 do not support adding caption tracks. 

![Image](https://github.com/user-attachments/assets/61f81e9a-aa48-4699-bc2e-edfee7463673)

Notice only media sources are available to add, while v1.6 cards allow adding tracks as well:

![Image](https://github.com/user-attachments/assets/65e201d4-990a-45b1-be5c-0359ad2b1614)


Transcript which should yield to video with captions (with newer adaptive cards supported):

```
[{
  "type": "message",
  "id": "ACxp7WbEh9b3DGQ8EGSjLF-us|0000001",
  "timestamp": "2025-02-11T19:34:24.643418Z",
  "channelId": "directline",
  "from": {
    "id": "hawo-mockbot4-mock-bot-bot",
    "name": "Bot",
    "role": "bot"
  },
  "conversation": {
    "id": "ACxp7WbEh9b3DGQ8EGSjLF-us"
  },
  "attachmentLayout": "carousel",
  "locale": "en-US",
  "text": "Showing  Productvideo",
  "attachments": [
    {
      "contentType": "application/vnd.microsoft.card.adaptive",
      "content": {
        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
        "type": "AdaptiveCard",
        "version": "1.1",
        "fallbackText": "This card requires Media to be viewed. Ask your platform to update to Adaptive Cards v1.1 for this and more!",
        "body": [
          {
            "type": "Media",
            "poster": "https://adaptivecards.io/content/poster-video.png",
            "altText": "Adaptive Cards overview video",
            "sources": [
              {
                "mimeType": "video/mp4",
                "url": "https://chrisdavidmills.github.io/html5-captions-and-subtitles-content-kit/demo/video/sintel-short.mp4"
              }
            ],
            "captionSources": [
                {
                    "mimeType": "captions",
                    "url": "https://chrisdavidmills.github.io/html5-captions-and-subtitles-content-kit/demo/final/vtt/sintel-captions-en.vtt",
                    "label": "English captions"
                }
            ]
          }
        ],
        "actions": [
          {
            "type": "Action.OpenUrl",
            "title": "Learn more",
            "url": "https://adaptivecards.io"
          }
        ]
      }
    }
  ],
  "replyToId": "ACxp7WbEh9b3DGQ8EGSjLF-us|0000000",
  "channelData": {
    "webChat": {
      "receivedAt": 1739302464688
    },
    "webchat:sequence-id": 1739302464643
  }
}]
```

In modern Web Chat this is solved by allowing arbitrary html tags directly into activity markup, so users can include video or audio elements without any issue.

**Comment by OEvgeny** [comment]
Closing as a third-party issue: Adaptive Cards of the version we use don't support Video Captions

**Comment by Kirank915** [comment]
@OEvgeny Thanks for the update. We will route this bug to third party team via eDAD team as per [External 3rd Party Process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2f%2fmicrosoft.sharepoint.com%2fteams%2fcea11y%2fSitePages%2fExternal-Bugs-Process---Third-Party-Dependencies.aspx&data=05%7c01%7cv-katkri%40microsoft.com%7c34f1c3338a5c4ca88a0908da4368ce41%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637896414685783444%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=2chKRW7U66oVfw0eCi6G7gu0n1acPjscrCUl6fyEi4A%3D&reserved=0). To route these, we might require some way (code pen or sample environment) where the external team will be able to repro the issue. 
We will route these bugs to eDAD team once we have the details.

**Comment by OEvgeny** [comment]
Please find the details above. open source version of Adaptive Cards doesn't support Video or Audio captions. Due to the fact that Web Chat is an open source project, we cannot rely on the new version of Adaptive Cards, which isn't OSS anymore.

We're already working with the team to resolve the issue.

**Comment by Kirank915** [comment]
@OEvgeny could you please confirm, if this issue will be fixed by your team or should we route this bug to third party team via eDAD team?
